app/2.0/docs/upgrade.md

```diff
@@ -973,7 +973,7 @@ Note that `Polymer.Element` provides a cleaner base class without much of the su
 
 
 ```
-this.dispatchEvent(new CustomEvent('some-event'), {bubbles: true});
+this.dispatchEvent(new CustomEvent('some-event', { bubbles: true }));
```

Comment:
FYI for the future our code style calls for omitting the spaces after an open curly brace, and before a closing curly brace. But less broken is more better. :D